proto/regen/ecocredit/marketplace/v1/types.proto

```diff
@@ -0,0 +1,24 @@
+syntax = "proto3";
+
+package regen.ecocredit.marketplace.v1;
+
+import "regen/ecocredit/marketplace/v1/state.proto";
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/regen-network/regen-ledger/x/ecocredit/marketplace";
+
+// AllowedDenomProposal is a gov Content type for approving a denom for use in
+// the marketplace.
+message AllowedDenomProposal {
+  option (gogoproto.goproto_stringer) = false;
+
+  // title is the title of the proposal.
+  string title = 1;
+
+  // description is the description of the proposal.
+  string description = 2;
+
+  // allowed_denom contains coin denom information that will be added to the
+  // list of allowed denoms for use in the marketplace.
+  AllowedDenom allowed_denom = 3;
```

Comment:
It's not allowed yet and might make more sense to name this field `denom`.

Reply:
[changed](https://github.com/regen-network/regen-ledger/pull/1072/commits/d701ec57ad94a754739581df1168d427fb24c47d#diff-1dd53158c058ff1c5c3df56edcef5901de3110192193488ddbc7a162a6bfce15R23)